core/rawdb/accessors_indexes.go

```diff
@@ -120,9 +121,9 @@ func ReadTransaction(db ethdb.Reader, hash common.Hash) (*types.Transaction, com
 
 // ReadReceipt retrieves a specific transaction receipt from the database, along with
 // its added positional metadata.
-func ReadReceipt(db ethdb.Reader, hash common.Hash, config *params.ChainConfig) (*types.Receipt, common.Hash, uint64, uint64) {
+func ReadReceipt(db ethdb.Reader, txHash common.Hash, config *params.ChainConfig) (*types.Receipt, common.Hash, uint64, uint64) {
```

Comment:
I think I checked all usages of rawdb.ReadReceipt and nowhere do we use the latter 3 return values, all calls look like the dogsled (`rcpt, _, _, _ := rawdb.ReadReceipt`). Maybe we can get rid of all return values except the receipt

Reply:
Yes, it looks like the dogsled 😄
I've checked, and it appears unnecessary now, so I'll delete them.
Thank you!